static/app/views/issueDetails/groupEventDetails/groupEventDetailsContent.tsx

```diff
@@ -124,6 +132,23 @@ function DefaultGroupEventDetailsContent({
           />
         </EventDataSection>
       )}
+      {event.type === EventOrGroupType.ERROR &&
+      organization.features.includes('ai-analytics') &&
+      event?.entries
+        ?.filter(x => x.type === EntryType.EXCEPTION)
+        ?.map(x => x.data as ExceptionType)
+        ?.flatMap(x => x.values)
+        ?.find(
+          x =>
+            (x?.value?.includes('API key') || x?.value?.includes('429')) &&
+            x?.value?.includes('openai')
+        ) ? (
```

Comment:
nit - there's an extra map  probably quite a few more `?` than needed. Try to only use them in the areas you expect things to not exist. Prefer .some to find for booleans
```suggestion
      event?.entries
        ?.filter((x): x is EntryException => x.type === EntryType.EXCEPTION)
        .flatMap(x => x.data.values)
        .some(
          x =>
            (x?.value?.includes('API key') || x?.value?.includes('429')) &&
            x?.value?.includes('openai')
        ) ? (
```